spec/models/translation_spec.rb

```diff
@@ -162,9 +162,7 @@
     it "uploads the translation to S3" do
       package = double
       allow(Package).to receive(:new).and_return(package)
-      allow(package).to receive(:push_to_s3)
-
-      translation.update(is_published: true)
+      allow(package).to receive(:push_published_to_s3)
```

Comment:
I believe this should still be `:push_to_s3` since it's on the Package class and not the Translation class